src/SourceBuild/content/repo-projects/templating.proj

```diff
@@ -9,8 +9,11 @@
 
   <ItemGroup>
     <RepositoryReference Include="arcade" />
```

Comment:
Templating has a runtime dependency (source-only and msft)

Reply:
```suggestion
    <RepositoryReference Include="arcade" />
    <RepositoryReference Include="runtime" />
```

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/bb5af848117833334b3756fa481aa6fbb077723f